flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -1024,9 +1024,25 @@ under the License.
 	<profiles>
 		<!-- Activate these profiles with -Phive-x.x.x to build and test against different Hive versions -->
 		<profile>
-			<id>hive-3.1.1</id>
+			<id>hive-3.1.2</id>
```

Comment:
Ideally we drop the patch version from the profile id so we don't have to touch the ci scripts every time we bump hive.

Reply:
I change it `hive3` now.